scim-sdk-client/src/main/java/de/captaingoldfish/scim/sdk/client/http/ScimHttpClient.java

```diff
@@ -68,9 +68,17 @@ public class ScimHttpClient implements Closeable
    */
   private CloseableHttpClient httpClient;
 
+  private HttpClientBuilder httpClientBuilder;
+
   public ScimHttpClient(ScimClientConfig scimClientConfig)
+  {
+    this(scimClientConfig, null);
+  }
+
+  public ScimHttpClient(ScimClientConfig scimClientConfig, HttpClientBuilder httpClientBuilder)
```

Comment:
Please let us add the customized httpClientBuilder with the scimClientConfig:

```java
ScimClientConfig.builder().httpClientBuilder(httpClientBuilder).build();
```

then we override the builder method:

```java
public class ScimClientConfig
{
    ...

    public HttpClientBuilder getHttpClientBuilder()
    {
      return this.httpClientBuilder == null ? HttpClientBuilder.create() : this.httpClientBuilder;
    }

    ...
}
```

With this we give even more control about the httpClient creation.

Reply:
Updated the commit as requested.